core/src/main/java/discord4j/core/spec/VoiceChannelJoinSpec.java

```diff
@@ -37,6 +38,10 @@
  */
 public class VoiceChannelJoinSpec implements Spec<Mono<VoiceConnection>> {
 
+    /** The default maximum amount of time in seconds to wait before the connection to the voice channel timeouts. */
+    private static final int DEFAULT_TIMEOUT = 5;
```

Comment:
Is this a good default? trying to clarify its magic

Reply:
Using `.elapsed()` to determine the average time of a connection, it seems to vary between 650ms and 1100ms on my personnal computer with optic fibre. If we assume an average connection time of 1s, a timeout of 5s can take into account network fluctuation and lags but it can maybe be reduced a little.